src/main/java/org/terasology/dynamicCities/construction/Construction.java

```diff
@@ -487,6 +515,78 @@ public boolean buildParcel(DynParcel dynParcel, EntityRef settlement, CultureCom
         return true;
     }
 
+    public RoadParcel.Status buildRoadParcel(RoadParcel parcel, EntityRef settlement) {
+        boolean containsRelevantRegion = false;
+        boolean segmentFailed = false;
+
+        // Factor by which the rect will be expanded while flattening
+        ImmutableVector2i rectExpansionFactor = new ImmutableVector2i(1, 1);
```

Comment:
What`s the meaning of the 1s passed as parameters? It would probably be better to make those numbers constant.